scripts/cover.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+mkdir -p cover
```

Comment:
you probably want to delete all files in the `cover` directory if it already exists